docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
 Column-based Noise
 ==================
 
+Column-based noise introduces errors in individual fields within a row of data.
+The types implemented in pseudopeople are listed below, along with instructions
+about how to configure them.
+
+.. contents::
+   :depth: 2
+   :local:
+
 Leave a field blank
 -------------------
 
+When this noise type occurs, whatever value was supposed to be collected is missing for this row.
+This could be because the input was left blank, an answer was refused,
+or the answer was illegible or unintelligible.
+pseudopeople will output a :code:`numpy.nan` value in
+the relevant cell.
+
+This noise type is called :code:`leave_blank` in the configuration. It takes one parameter:
+
+.. list-table:: Parameters to the :code:`leave_blank` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that a cell in the column being configured is blank.
+      This probability is the same for all cells in a column;
+      missingness is not correlated with any attributes.
+    - 0.01 (1%)
+
 Choose the wrong option
 -----------------------
 
-Write the wrong zipcode digits
-------------------------------
+Some survey questions or administrative forms ask the respondent to choose
+between a list of options.
+This noise type occurs when the wrong option is chosen by mistake.
```

Comment:
```suggestion
This noise type occurs when the wrong option is chosen, either intentionally or by mistake.
```

Reply:
Actually, it's confusing to start this sentence with "This noise type" because the previous sentence was not talking about a noise type. The paragraph structure for "choose the wrong option" and "leave a field blank" is a bit different from the other two noise functions, so I think they should be edited to all match. Trying to come up with some suggested edits now...

Reply:
Here's one idea:

If a question on a survey or administrative form provides a list of options, respondents may sometimes choose the wrong option, either intentionally or by mistake. pseudopeople simulates this type of noise by selecting from the available options uniformly at random for a fraction of the cells in a categorical column. The [Datasets](https://github.com/ihmeuw/pseudopeople/pull/datasets/index.html#datasets-main) page lists the available options for each categorical column in pseudopeople's datasets.

Reply:
Thanks for this -- it's much better! I'm revising it a bit further to make it less statistical-sounding, and because I don't think "categorical" in this context will reflect how our users think about things.